### PR TITLE
Move availableJobs to Config | PR Template

### PR DESCRIPTION
**Pull request by Sharkiiie**
## Description

1. Moves jobs available in city hall menu to config to make it easier for newbies to find.
1.5 (I am the professional config maker)
2. Updates PR template.

## Checklist

-   [ ] I have personally loaded this code into an updated qbcore project and checked all of its functionality.
(no, fivem is down 🤪)
-   [x] My code fits the style guidelines.
-   [x] My PR fits the contribution guidelines.

### CODE REVIEW

**Comment by r0adra93** [comment]
code works! LGTM MERGE IT!

**Review by r0adra93 — APPROVED** [review]
code works! LGTM MERGE IT!

File: server/main.lua [hunk 1]
```diff
@@ -115,14 +108,15 @@ RegisterNetEvent('qb-cityhall:server:ApplyJob', function(job, cityhallCoords)
...
+
     local data = {
         ["src"] = src,
         ["job"] = job
     }
     if #(pedCoords - cityhallCoords) >= 20.0 or not availableJobs[job] then
         return false -- DropPlayer(source, "Attempted exploit abuse")
     end
-    if QBCore.Shared.QBJobsStatus then exports["qb-jobs"]:submitApplication(data)
+    if QBCore.Shared.QBJobsStatus then exports["qb-jobs"]:submitApplication(data,"Jobs")
```
Comment (zwkf) [inline server/main.lua:119]: Remember to put a space before passing the second param

Comment (Sharkiiie) [reply server/main.lua:119]: I blame road, I'll fix it rn

Comment (r0adra93) [reply server/main.lua:119]: It's a space not necessary

**Review by zwkf — APPROVED** [review]
LGTM

**Pull request merged**